traits.yml

```
@@ -73,6 +73,43 @@ traits:
 
               This must be a valid URL so special characters need to
               be encoded.
+  identity:
+    description: Traits that aid identification of an entity
+    members:
+      DisplayName:
+        description: >
+          Names that can be presented to a user in order to identify
+          and/or disambiguate the entity. These strings are potentially
+          unstable and should not be used as a UUID or other persistent
+          anchor.
+        usage:
+          - entity
+        properties:
+          name:
+            type: string
+            description: |
+              The humanised name of entity itself, not including any
```

[Consider][Minor]
```suggestion
              The humanized name of entity itself, not including any
```

Came up as a squiggle in my editor because of the UK spelling. Hesitate to bring this up as the Americanisation  by default bugs me. We mention in the readme that we should write US english copy on the decision records, but I can't find a reference anywhere else, so I don't know if it's part of our standards for all doc.